index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HTML Homework</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <style>
+        body {
+            margin: 0;
+            color: #ffeeaa;
+            font-weight: 700;
+            line-height: 2;
+            font-size: calc(14px + (30 - 14) * ((100vw - 300px) / (1600 - 300)));
+        }
+
+        .grid-container {
+            display: grid;
+            grid-template-areas: 'l l n n n n n n n n n n' 'a a m m m m m m m m m m' 'f f f f f f f f f f f f';
+            grid-template-columns: 205px auto;
+            grid-template-rows: 60px auto 60px;
+            grid-gap: 5px;
+            background-color: #ffeeaa;
+            padding: 20px;
+            min-height: calc(100vh - 40px);
+        }
+        .logo {
+            grid-area: l;
+            background: #92cab3;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+        nav {
+            grid-area: n;
+            background: #ff706a;
+            display: flex;
+            align-items: center;
+        }
+        nav > span:not(:last-child) {
+            margin: 0 20px;
+        }
+        nav > span:last-child {
+            margin: 0 20px 0 auto;
+        }
+        aside {
+            grid-area: a;
+            background: #00ca97;
+            display: flex;
+            flex-direction: column;
+        }
+        aside > span:first-child {
+            margin-bottom: auto;
+        }
+        footer {
+            grid-area: f;
+            background: #dc9cdd;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        main {
+            grid-area: m;
+            background: #fbca5a;
+            display: grid;
+            grid-template-areas: 'c1 c1 c1 c2 c2 c2 c2 c2 c4 c4 c4 c4' 'c3 c3 c3 c3 c3 c3 c3 c3 c4 c4 c4 c4';
```

Comment:
You're missing `grid-template-rows` and `grid-template-columns` properties for screens > 767px